### PR TITLE
Health fixes

### PR DESCRIPTION
**Pull request by radarhere**

### CODE REVIEW

File: PIL/ImageDraw.py [hunk 1]
```diff
@@ -267,7 +268,7 @@ def multiline_text(self, xy, text, fill=None, font=None, anchor=None,
         lines = self._multiline_split(text)
         line_spacing = self.textsize('A', font=font)[1] + spacing
         for line in lines:
-            line_width, line_height = self.textsize(line, font)
```
Comment (wiredfool) [inline PIL/ImageDraw.py]: This is clearer as currently written

File: PIL/ImageDraw.py [hunk 2]
```diff
@@ -297,11 +298,10 @@ def textsize(self, text, font=None):
 
     def multiline_textsize(self, text, font=None, spacing=0):
         max_width = 0
-        height = 0
         lines = self._multiline_split(text)
         line_spacing = self.textsize('A', font=font)[1] + spacing
         for line in lines:
-            line_width, line_height = self.textsize(line, font)
```
Comment (wiredfool) [inline PIL/ImageDraw.py]: This is also clearer as currently written.

File: PIL/ImageDraw.py (hunk 1 above)
Comment (radarhere) [inline PIL/ImageDraw.py]: Okay, I have removed this change.

File: PIL/ImageDraw.py (hunk 2 above)
Comment (radarhere) [inline PIL/ImageDraw.py]: I have also removed this change.

**Pull request merged**